test/e2e/parallel/proxy_test.go

```diff
@@ -969,6 +984,24 @@ func TestSpaceLister(t *testing.T) {
 		require.NoError(t, err)
 	})
 
+	t.Run("banned user cannot get workspace", func(t *testing.T) {
+		// when
+		workspace, err := users["banneduser"].getWorkspace(t, hostAwait, users["banneduser"].compliantUsername)
+
+		// then
+		require.EqualError(t, err, "the server could not find the requested resource (get workspaces.toolchain.dev.openshift.com banneduser)")
+		assert.Empty(t, workspace)
```

Comment:
shouldn't we use `require.Empty` instead?

```suggestion
		require.Empty(t, workspace)
```

Reply:
Or `require.Nil`?
```suggestion
		require.Nil(t, workspace)
```

Reply:
thanks!